### DISCUSSION TITLE
Add debug mode for API/live timing

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#241] [Feature requests] [answered]
Add a mode where you can push fake live timing data through the API for test/debug purposes.

For example a button to push a fake yellow flag, VSC/SC or safety car.

**Comment by ManCaveMedia** [comment]
It's quite easy to open the Live Timing Replay of a race with yellow, double yellow and red flags as is. So I'm not really sure what this would add.

**Comment by LapsTimeOFF** [comment, marked as answer]
Since this is more headed towards developers this will likely not be added.
